aws-observability/apps/apigateway/api_gateway_app.template.yaml

```diff
@@ -203,6 +203,40 @@ Resources:
       SumoAccessKey: !Ref Section1cSumoAccessKey
       SumoDeployment: !Ref Section1aSumoDeployment
 
+  AccessLogsFieldExtractionRule:
+    Type: Custom::SumoLogicFieldExtractionRule
+    Condition: create_fer
+    DependsOn: 
+      - AddApiNameField
+      - AddApiIdField
+    Properties:
+      ServiceToken: !Ref Section4aParentStackLambdaARN
+      RemoveOnDeleteStack: false
+      FieldExtractionRuleName: "AwsObservabilityApiGatewayAccessLogsFER"
+      FieldExtractionRuleScope: "account=* region=* apiId domainName stage requestId status"
+      FieldExtractionRuleParseExpression: '| json "apiId", "domainName", "stage" as apiId, domainName, stage
+                                           | "aws/apigateway" as namespace
+                                           | apiId as apiNames
+                                           | fields apiName, namespace, apiId'
+      FieldExtractionRuleParseEnabled: true
+      SumoAccessID: !Ref Section1bSumoAccessID
+      SumoAccessKey: !Ref Section1cSumoAccessKey
+      SumoDeployment: !Ref Section1aSumoDeployment
+
+  MetricRule:
+    Type: Custom::SumoLogicMetricRules
+    DependsOn: AddLoadBalancerField
+    Properties:
+      ServiceToken: !Ref Section4aParentStackLambdaARN
+      RemoveOnDeleteStack: false
+      MetricRuleName: "AwsObservabilityApiGatewayApiNameEntityRule"
```

Comment:
See if you can have consistent name for all metric rule
**AwsObservability**ApiGatewayApiName**MetricsEntityRule**